tests/jax/models/Roberta/test_roberta.py

```diff
@@ -0,0 +1,91 @@
+# SPDX-FileCopyrightText: (c) 2024 Tenstorrent AI ULC
+#
+# SPDX-License-Identifier: Apache-2.0
+
+from typing import Dict, Sequence
+
+import jax
+import pytest
+from flax import linen as nn
+from infra import ComparisonConfig, ModelTester, RunMode
+from transformers import AutoTokenizer, FlaxRobertaForMaskedLM
+
+MODEL = "FacebookAI/roberta-base"
+
+# ----- Tester -----
+
+
+class FlaxRobertaForMaskedLMTester(ModelTester):
+    """Tester for DistilBert model with a `language modeling` head on top."""
+
+    # @override
+    @staticmethod
+    def _get_model() -> nn.Module:
+        return FlaxRobertaForMaskedLM.from_pretrained(MODEL)
+
+    # @override
+    @staticmethod
+    def _get_forward_method_name() -> str:
+        return "__call__"
+
+    # @override
+    @staticmethod
+    def _get_input_activations() -> Sequence[jax.Array]:
+        tokenizer = AutoTokenizer.from_pretrained(MODEL)
+        inputs = tokenizer("Hello <mask>.", return_tensors="np")
+        return [inputs["input_ids"]]
+    
+    # @override
+    def _get_forward_method_args(self):
+        return self._get_input_activations()
+
+    # @override
+    def _get_forward_method_kwargs(self) -> Dict[str, jax.Array]:
+        assert hasattr(self._model, "params")
+        return {"params": self._model.params}
+
+    # @ override
+    def _get_static_argnames(self):
+        return ['train']
+
+
+# ----- Fixtures -----
+
+
+@pytest.fixture
+def comparison_config() -> ComparisonConfig:
+    config = ComparisonConfig()
+    config.disable_all()
+    config.pcc.enable()
+    return config
```

Comment:
It is ok if some model passes only one metric, it can happen. But since you skip both tests down below (for now) you can remove this completely since we can't know if it passes all other metrics as well.

Reply:
good catch, it was indeed a remnant of older versions of the infra